app/views/posts/index.html.erb

```diff
@@ -1,10 +1,10 @@
+<div class='post-index'>
 <h1>All posts</h1>
 
-<ul>
-
-    <% @posts.each do |post| %>
-    
-        <li><%= link_to post.title, user_posts_path(post) %></li>
-    
+<div>
+    <% @posts.each do |post| %> 
+        <h2><%= link_to post.title, user_post_path(user_id: post.author.id, id: post.id) %></h2>
+        <p><%= post.text %></p>
     <% end %>
```

Comment:
- [ ] It would be better to add comments by using the method `recents_comments` that you created in the last task
to match the wireframe and also complete the requirements